source/_integrations/onedrive.markdown

```
@@ -0,0 +1,38 @@
+This integration allows you to use [OneDrive](https://www.microsoft.com/en-us/microsoft-365/onedrive/online-cloud-storage) for use with Home Assistant Backups.
+Backups will be created in the `App Folder` of your OneDrive, the integration does have no access to any other parts of your OneDrive except its own folder.
+
+{% include integrations/config_flow.md %}
+
+## Getting application credentials
+
+{% note %}
+You will need an Azure tenant with an active Azure subscription to create your own client credentials.
+{% endnote %}
+
+This integration comes with a predefined set of [application credentials](https://www.home-assistant.io/integrations/application_credentials/) through Home assistant account linking. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)